src/status_im/contexts/wallet/collectible/tabs/overview/view.cljs

```diff
@@ -0,0 +1,72 @@
+(ns status-im.contexts.wallet.collectible.tabs.overview.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.collectible.tabs.overview.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- traits-section
+  []
+  (let [traits (rf/sub [:wallet/last-collectible-details-traits])]
+    (when (pos? (count traits))
+      [rn/view
+       [quo/section-label
+        {:section         (i18n/label :t/traits)
+         :container-style style/traits-title-container}]
+       [rn/flat-list
+        {:render-fn               (fn [{:keys [trait-type value]}]
+                                    [quo/data-item
+                                     {:description     :default
+                                      :card?           true
+                                      :status          :default
+                                      :size            :default
+                                      :title           trait-type
+                                      :subtitle        value
+                                      :container-style style/traits-item}])
+         :data                    traits
+         :key                     :collectibles-list
+         :key-fn                  :id
+         :num-columns             2
+         :content-container-style style/traits-container}]])))
+
+(defn- info
+  []
+  (let [chain-id        (rf/sub [:wallet/last-collectible-details-chain-id])
+        network         (rf/sub [:wallet/network-details-by-chain-id
+                                 chain-id])
+        network-keyword (get network :network-name)
+        network-name    (string/capitalize (name network-keyword))]
+    [rn/view
+     {:style style/info-container}
+     [rn/view {:style style/account}
+      [quo/data-item
+       {:description         :account
+        :card?               true
+        :status              :default
+        :size                :default
+        :title               (i18n/label :t/account-title)
+        :subtitle            "Collectibles vault"
```

Comment:
No translation on purpose?

Reply:
This part is not part of the ticket and it's still a mock data.

Reply:
It was mock data before. I added it as a dynamic one now.